vector/src/main/java/im/vector/app/features/home/room/detail/timeline/action/MessageActionsViewModel.kt

```diff
@@ -333,6 +334,13 @@ class MessageActionsViewModel @AssistedInject constructor(
     ) {
         val eventId = timelineEvent.eventId
         if (!timelineEvent.root.isRedacted()) {
+            if (initialState.isFromPinnedMessagesTimeline) {
+                if (actionPermissions.canPinMessage && vectorPreferences.arePinnedMessagesEnabled()) {
+                    add(EventSharedAction.UnpinMessage(eventId))
+                    add(EventSharedAction.ViewPinnedMessageInRoom(eventId))
```

Comment:
The action ViewPinnedMessageInRoom should be available even if the user cannot pin and unpin message.